.github/workflows/pysat_rc.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test with latest pysat RC
+
+on: [workflow_dispatch]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Is there a TODO for windows support?

Reply:
So, I added windows CI tests to the other active PR and they broke.  Issue #76 has been created to capture the problem.

Reply:
Do I need to update anything here for the windows issues?

Reply:
Just add the TODO for #76.